scripts/es2k/setup_infra.sh

```diff
@@ -62,13 +52,13 @@ function install_drivers () {
   modprobe mdev
   modprobe vfio-pci
   modprobe vfio_iommu_type1
-  # change this with insmod in case of new idpf driver
+  # change this with insmod in case of new idpf host driver
   modprobe idpf
 }
 
-# Get PCI device ID for es2k
+# Get PCI device ID for IDPF
 function get_device_id () {
-  dev_id=$(lspci | grep 1453 | cut -d ':' -f 1)
+  dev_id=$(lspci | grep 1452 | cut -d ':' -f 1)
```

Comment:
Is this correct? Maybe we should add a comment explaining what are 1452 and 1453 each

Reply:
This is correct. Both would give the same bus number but 1452 is correct